learn/getting_started/meilisearch_101.md

```diff
@@ -0,0 +1,12 @@
+# MeiliSearch 101
```

Comment:
I want this page to be an overview of the 101, talk about what the user should expect, who its for, any prerequisites, etc. A lot of documentation websites have this. I can try improving it otherwise we can get rid of it.